### ISSUE TITLE
ABLUnit - OE11.7.8 change of behavior

### ISSUE THREAD

**Issue by clement-brodu** [Riverside-Software/pct#435]
### Subject of the issue
Behavior change in 11.7.8 and broke a unit test : Ablunit Test8.

I move from 11.7.4 to 11.7.8 and now this unit test failed. It seems the change made in 12.2 has been report in 11.7.8 (or 11.7.7 or 11.7.6 or 11.7.5).

### Your environment
* Operating system and version : Windows 10
* OpenEdge version : 11.7.8
* Ant version : 1.10.5
* PCT version : 215

### Actual behaviour

Ablunit Test8 fail because the build doesn't fail.

```

Created dir: E:\Outils\Jenkins\workspace\Indus_PCT_feat_compile_tempdir\testbox\ABLUnit\test8\tempUnable to build type info  
for E:\Outils\Jenkins\workspace\Indus_PCT_feat_compile_tempdir\testbox\ABLUnit\test8\src\ABLUnitTest.cls: Could not locate  
element 'isTue' in class 'OpenEdge.Core.Assert'. (12927)QUIT statement foundTotal tests run: 0, Failures: 0, Errors: 0 

--

java.lang.AssertionError: Should throw BuildException because: Syntax error at   
com.phenix.pct.BuildFileTestNg.expectSpecificBuildException(BuildFileTestNg.java:103) at   
com.phenix.pct.BuildFileTestNg.expectBuildException(BuildFileTestNg.java:81) at   
com.phenix.pct.ABLUnitTest.test8(ABLUnitTest.java:131) ... Removed 25 stack frames  

```
### Expected behaviour
Unit test should be ok

### Steps to reproduce
Execute the unit test with a 11.7.8 version. 

Be sure to have a full installation or a netsetup installation.

**Comment by gquerret** [comment]
I thought I reported that as a problem to tech support, but can't find any mention of it...
This change means that test classes that can't be compiled are discarded from the ABLUnit report. Nobody will ever notice the entry in the Jenkins log file.

I'll introduce the same workaround for 11.7.8 as for 12.2.

**Comment by gquerret** [comment]
Test is skipped on 11.7.8
Issue reported to Progress tech support.

**Comment by gquerret** [comment]
https://knowledgebase.progress.com/articles/Article/ABLUnit-regression-in-OpenEdge-11-7-8-and-12-2?popup=true

**Comment by clement-brodu** [comment]
It's seems that a fix is available in 12.2.4 : OCTA-32162

https://docs.progress.com/fr-FR/bundle/openedge-product-notes/page/Issues-fixed-in-OpenEdge-12.2.4.html

**Comment by gquerret** [comment]
I'll install this patch on the build server and verify that. Thanks for the notification.
